### DISCUSSION TITLE
Error: Response not successful: Received status code 400

### DISCUSSION THREAD

**Discussion by dhawansolanki** [RakeshPotnuru/Blog#10] [Q&A]
Trying to recreate a Blog Website..
EVERYTHING Works fine except the slug post part..

I am able to connect to the Api and able to fetch the posts data..
All the posts appears on home page (link 1)but when I click on the post to see more about it.. the following error appears(Link 2)

I Have attached the screenshot of both
link 1 : Home page where all the post appears
![d1](https://user-images.githubusercontent.com/91565429/187688708-2fd9193c-fad9-4e35-b4b6-5d16ccec4c5f.png)

link2 : Post page after clicking on link
![d2](https://user-images.githubusercontent.com/91565429/187688823-8b182ae6-0466-46b1-9002-d745af31a042.png)

Please Do Help. Thank youu.

**Comment by RakeshPotnuru** [comment]
@dhawansolanki 
Can you show errors in the console log?
Also, have you edited any part of the source code?

**Reply by dhawansolanki** [reply]
![err1](https://user-images.githubusercontent.com/91565429/187744218-0daf1416-20eb-4660-aef2-f1313173bbe2.PNG)

Console log for the above said error.

Also I have made some changes in UI Styles.. not in any backend stuffs.
For your reference i have attached my code repo :
https://www.github.com/dhawansolanki/B

Its Really Urgent. 
Thankyou.

**Reply by RakeshPotnuru** [reply]
@dhawansolanki
Can you try setting up without modifying any source code? See if you can set up or not.

**Reply by dhawansolanki** [reply]
@RakeshPotnuru 
Yes At first i tried without making any changes.. though it didnt worked..
So made some changes..

Could you please share the exact code files(everything included except the hygraph credintials?)..

**Reply by RakeshPotnuru** [reply]
This is the exact codebase. Everything included.
Have you entered the correct env variables?

These are must:

```
NEXT_PUBLIC_HYGRAPH_CONTENT_API_URL=
NEXT_PUBLIC_HYGRAPH_AUTH_TOKEN=
NEXT_PUBLIC_CLOUDINARY_CLOUDNAME=
```

And make sure you named env file as `.env.local`

**Reply by dhawansolanki** [reply]
Yesss my env.local was already correct.
Still the error persists..
Hiw Do we need install cloudinary in hygraph.. i didn't understood the doc..

**Reply by RakeshPotnuru** [reply]
Go to project settings -> UI Extensions -> Add UI Extension -> Put the extension URL -> Click "Run compatibility test" -> Then fill secret variables.

**Extension URL:**

- Run this command to download the extension code.
```bash
npx degit hygraph/hygraph-examples/uix-cloudinary-input uix-cloudinary-input
```

- Create a GitHub repo for this extension and push code.
- Host extension on Vercel.
- After that, you will get the extension URL.

**Reply by dhawansolanki** [reply]
Okaayy Thank you
